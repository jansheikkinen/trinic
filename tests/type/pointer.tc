// Pointer

function main(void) -> void do
  let a: *int8 = undefined;
end
