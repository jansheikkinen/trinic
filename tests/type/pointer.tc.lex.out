
### LEXER ###
000:     FUNCTION,     (null), (003, 008)
001:   IDENTIFIER,       main, (003, 013)
002:            (,     (null), (003, 014)
003:         VOID,     (null), (003, 018)
004:            ),     (null), (003, 019)
005:           ->,     (null), (003, 021)
006:         VOID,     (null), (003, 026)
007:           DO,     (null), (003, 029)
008:          LET,     (null), (004, 005)
009:   IDENTIFIER,          a, (004, 007)
010:            :,     (null), (004, 008)
011:            *,     (null), (004, 010)
012:         INT8,     (null), (004, 014)
013:            =,     (null), (004, 016)
014:    UNDEFINED,     (null), (004, 026)
015:            ;,     (null), (004, 027)
016:          END,     (null), (005, 003)
